types/multi-progress/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "adjacent-overload-signatures": false,
```

Comment:
Why are the lint rules disabled?

Reply:
Again, my mistake for copying these without checking whether they were actually needed. Fixed.